Lib/test/test_c_locale_coercion.py

```diff
@@ -160,7 +147,7 @@ def get_child_details(cls, env_vars):
         if not result.rc == 0:
             result.fail(py_cmd)
         # All subprocess outputs in this test case should be pure ASCII
-        adjusted_output = cls._handle_output_variations(result.out)
+        adjusted_output = result.out.lower()
```

Comment:
What is the purpose of .lower()?

Reply:
It was in the removed function (under those replaces) because according to the comment: "_Some platforms report UTF-8 instead of utf-8_". But maybe that is also fixed by guaranteed normalization?

Also when I remove it, tests start failing, because 'os.environ.get()' functions return C and the suite expects c (as everything was lowered before), but I can fix that as well.

Reply:
> It was in the removed function (under those replaces) because according to the comment: "Some platforms report UTF-8 instead of utf-8". But maybe that is also fixed by guaranteed normalization?

That's the purpose of normalization: always get the same string :-)

```
>>> codecs.lookup("UTF8").name
'utf-8'
>>> codecs.lookup("UTF-8").name
'utf-8'
>>> codecs.lookup("UTF_8").name
'utf-8'
>>> codecs.lookup("utf8").name
'utf-8'
```

> Also when I remove it, tests start failing, because 'os.environ.get()' functions return C and the suite expects c (as everything was lowered before), but I can fix that as well.

Please use "C" instead of "c" :-)

Reply:
> That's the purpose of normalization: always get the same string :-)

Well, I could have thought of that...

> Please use "C" instead of "c" :-)

Fixed